### PR TITLE
adds links to README

### PR DESCRIPTION
**Pull request by ksatirli**
This PR adds links to the top-most README to make it easier to find what's going on.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,13 +6,16 @@
...
   - [License](#license)
 
-## Reading material
+## Examples
 
-This section is a collection of links that will help you make the most of today's session.
+* [CLI Enhancement: Provider Lock files](https://github.com/ksatirli/whats-new-with-terraform-014/tree/main/terraform-init)
+* [Core Enhancement: Mark variables as sensitive](https://github.com/ksatirli/whats-new-with-terraform-014/tree/main/sensitive-variables)
+* [Core Enhancement: Reading and Writing State files from multiple versions](https://github.com/ksatirli/whats-new-with-terraform-014/tree/main/reading-state-files)
+* [Enhancements: `alltrue and `anytrue`](https://github.com/ksatirli/whats-new-with-terraform-014/tree/main/alltrue-anytrue)
```
Comment (onlydole) [inline README.md:18]:
```suggestion
* [Enhancements: `alltrue` and `anytrue`](https://github.com/ksatirli/whats-new-with-terraform-014/tree/main/alltrue-anytrue)
```

Comment (onlydole) [reply README.md:18]: Small fix: updating ` for wrapping the formatting on this line

**Review by onlydole — CHANGES_REQUESTED** [review]
A small nit, otherwise this looks great!

**Comment by ksatirli** [comment]
Thanks @onlydole!

**Pull request merged**